src/internal/m365/collection/site/collection.go

```diff
@@ -26,14 +26,21 @@ import (
 
 type DataCategory int
 
+// channel sizes
+const (
+	collectionChannelBufferSize = 50
+	fetchChannelSize            = 5
+)
+
 //go:generate stringer -type=DataCategory
 const (
-	collectionChannelBufferSize              = 50
-	fetchChannelSize                         = 5
-	Unknown                     DataCategory = iota
-	List
-	Drive
-	Pages
+	// Unknown is 2 due to a mishandling of iota.
+	// It should be 0, by standard, but now we're
+	// stuck with this for backwards compatibility.
+	Unknown DataCategory = 2
+	List    DataCategory = 3
+	Drive   DataCategory = 4
+	Pages   DataCategory = 5
```

Comment:
These are never persisted. It doesn't matter what their values are since there's no backwards compatibility issues we could encounter. They [appear](https://github.com/alcionai/corso/blob/8d3fdeeb8dbd8dc68284c1d33b0b431d309b4f5b/src/internal/m365/collection/site/collection.go#L200) to only be used to determine what data to retrieve (and `Drive` doesn't seem to be used at all?)

Reply:
mmkay, I removed them entirely in favor of path.CategoryTypes.